frappe/modules/patch_handler.py

```diff
@@ -67,6 +67,8 @@ def run_patch(patch):
 				raise
 			else:
 				print("Failed to execute patch")
+				if skip_failing:
+					update_patch_log(patch, skipped=True)
```

Comment:
is we're *only* calling this on skip_failing - won't this be inaccurate? as we're not logging the patches which were run properly

Reply:
should be something like this ?

```suggestion
				update_patch_log(patch, skipped=skip_failing)
```